drivers/regulator/regulator_pca9420.c

```diff
@@ -465,6 +488,8 @@ static int regulator_pca9420_common_init(const struct device *dev)
 		.vin_ilim_ua = DT_INST_PROP(inst, nxp_vin_ilim_microamp),      \
 		.enable_modesel_pins =                                         \
 			DT_INST_PROP(inst, nxp_enable_modesel_pins),           \
+		.asys_uvlo_sel_mv =                                            \
+			DT_INST_PROP(inst, nxp_asys_uvlo_sel_millivolt),       \
```

Comment:
```suggestion
		.asys_uvlo_sel =                                            \
			DT_INST_ENUM_IDX(inst, nxp_asys_uvlo_sel_millivolt),       \
```

Reply:
Thank you for the review, and suggestion! Did not know about DT_INST_ENUM_IDX.